var/spack/repos/builtin/packages/py-protobuf/package.py

```diff
@@ -62,7 +62,7 @@ class PyProtobuf(PythonPackage):
     variant("cpp", default=False, when="@:4.21", description="Enable the cpp implementation")
 
     depends_on("python", type=("build", "link", "run"))
-    depends_on("py-setuptools", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
```

Comment:
Setuptools is required at run-time and is imported here: https://github.com/protocolbuffers/protobuf/blob/main/python/google/__init__.py